frontend/src/views/PurchasesSummary.vue

```diff
@@ -99,20 +142,26 @@ import { lastYear, diagnosticYears, normaliseText, getPercentage, getSustainable
 import BreadcrumbsNav from "@/components/BreadcrumbsNav"
 import DsfrSelect from "@/components/DsfrSelect"
 import DsfrAutocomplete from "@/components/DsfrAutocomplete"
+import DsfrTextField from "@/components/DsfrTextField"
 import FamiliesGraph from "@/components/FamiliesGraph"
 import labels from "@/data/quality-labels.json"
+import validators from "@/validators"
 
 export default {
   name: "PurchasesSummary",
-  components: { BreadcrumbsNav, DsfrSelect, DsfrAutocomplete, FamiliesGraph },
+  components: { BreadcrumbsNav, DsfrSelect, DsfrAutocomplete, DsfrTextField, FamiliesGraph },
   data() {
     return {
       vizYear: lastYear(),
+      vizCanteenId: null,
       vizCanteen: null,
       allowedYears: diagnosticYears().filter((year) => year <= lastYear() + 1),
       summary: null,
       loading: false,
       labels,
+      validators,
+      newYearlyMealCount: undefined,
```

Comment:
I usually have a preference of setting these to `null` (because technically they are defined) but it's only a matter of preference.